### ISSUE TITLE
Q: continue to update to latest or same tag

### ISSUE THREAD

**Issue by circa10a** [circa10a/ouroboros#43]
After getting a little bit of feedback, some folks are concerned about ouroboros auto updating containers to use the latest tag. It is sometimes considered "destructive" for applications that receive a major version release.

My question is, do the contributors here think ouroboros should continue to update to the latest tag or just check for updates on the current tag being used? For example if a container is running nginx:1-15 and nginx releases 1.15.1 and retags 1-15 to use the new minor patch,  should ouroboros just update using that new image, but same tag?

@tlkamp @dannysauer

**Comment by tlkamp** [comment]
Por que no los dos?

Maybe the default is to go to latest, but can be overridden by the user if they want to keep the current tag?

**Comment by circa10a** [comment]
Maybe a `--keep-tag` option would be a good way to about it. Good thought. I like it.

**Comment by tlkamp** [comment]
Yeah, this is a kind of feature that, if not given the option to change, will segment your consumer base. We want everyone! Customize all the things!
